web/docs/auth/overview.md

```diff
@@ -200,7 +201,7 @@ page AccountPage {
 ```
 
 ```tsx title="client/pages/Account.tsx"
-import type { User } from '@wasp/entities'
+import type { User } from '@wasp/auth/types'
```

Comment:
We can rename the `User` to `AuthenticatedUser` to keep the distinction between the Prisma user with `providerData` as a `string` and the `User` with deserialized `providerData` 🤷

Reply:
I did this for now:
```ts
import { User as AuthenticatedUser } from '@wasp/auth/types'
```

Reply:
Ha ok, I would prefer the opposite, because it would inform them about what that User really is, but this does help a bit I guess.
I understand of course that it requires bigger refactoring.